Library/Homebrew/cmd/pull.rb

```diff
@@ -130,12 +217,33 @@ def pull
         ohai "Patch closes issue ##{issue}"
         message = `git log HEAD^.. --format=%B`
 
-        if ARGV.include? "--bump"
-          odie "Can only bump one changed formula" unless changed_formulae.length == 1
+        if do_bump
+          odie "cannot bump: no changed formulae found after applying patch" if changed_formulae.empty?
           formula = changed_formulae.first
-          subject = "#{formula.name} #{formula.version}"
+          ohai "stable: #{formula.stable}"
+          new_versions = {
+              stable: formula.stable.nil? ? nil : formula.stable.version.to_s,
+              devel: formula.devel.nil? ? nil : formula.devel.version.to_s
+          }
+          subject_strs = []
+          if old_versions[:stable] != new_versions[:stable]
+            if new_versions[:stable].nil?
+              subject_strs << "remove stable"
+            else
+              subject_strs << "#{formula.version}"
+            end
+          end
+          if old_versions[:devel] != new_versions[:devel]
+            if new_versions[:devel].nil?
+              subject_strs << "remove devel"
```

Comment:
I'm kind of "eh" about notifying people we've removed devel in favour of stable. It happens a lot and feels obvious that sometimes there won't be a `devel` release available if stable is newer than it.